src/libAtomVM/bif.c

```diff
@@ -861,6 +861,35 @@ term bif_erlang_mul_2(Context *ctx, uint32_t fail_label, int live, term arg1, te
     }
 }
 
+term bif_erlang_fdiv_2(Context *ctx, uint32_t fail_label, int live, term arg1, term arg2)
```

Comment:
This is for dynamically running `a / b`. It's written for our needs and doesn't support boxed integers. I can open an issue for fixing it and aligning with other operators here but I'd like to merge it in this form if possible.

Reply:
You don't have to worry about boxed integers here: `term_conv_to_float()` does all the magic (and it handles boxed integers for you) :) So in this case you can replace the entire check with:
```c
VALIDATE_VALUE_BIF(fail_label, arg1, term_is_number);
VALIDATE_VALUE_BIF(fail_label, arg2, term_is_number);
```